autoapi/mappers/base.py

```diff
@@ -96,6 +98,27 @@ def short_name(self):
         return self.name.split('.')[-1]
 
     @property
+    def pathname(self):
+        '''Sluggified path for filenames
+
+        Slugs to a filename using the follow steps
+
+        * Decode unicode to approximate ascii
+        * Remove existing hypens
+        * Substitute hyphens for non-word characters
+        * Break up the string as paths
+        '''
+        slug = self.name
+        try:
+            slug = self.name.split('(')[0]
+        except IndexError:
+            pass
+        slug = unidecode.unidecode(slug)
+        slug = slug.replace('-', '')
```

Comment:
Is this to remove collision possibilities? It seems like we might want to keep them around.